src/EmbeddedVsix/EmbeddedVsix.csproj

```diff
@@ -1,12 +1,13 @@
-﻿<?xml version="1.0" encoding="utf-8"?>
-<Project ToolsVersion="15.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+﻿<Project>
```

Comment:
I updated the project to be SDK-style, based on the contents of the Integration.VSIX project. Specifically:

change to use the SDK imports
deleted unnecessary properties and item groups
copied the appropriate settings from the Integration.VSIX project